src/js/jsx/sections/style/AppearancePanel.jsx

```diff
@@ -152,13 +154,23 @@ define(function (require, exports, module) {
                 "section-container": true,
                 "section-container__collapsed": !this.props.visible
             });
-            
+
+            var selectedLayers = this.props.document.layers.selected,
+                mixedVectorType = selectedLayers.some(function (layer) {
```

Comment:
`mixedVectorType` does not sound like a boolean. Maybe call it `hasMixedVectorType`?